### PR TITLE
Game Session

### PR DESCRIPTION
**Pull request by cmonteiro128**
- Added the ability to start a game session
- Checks game session on app load
- Created button to initiate session

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,32 +1,90 @@
...
+  {
+    tabBarComponent: TabBarBottom,
+    tabBarPosition: 'bottom',
+  },
+);
+
+/** End Routes */
 
 /**
  * Test function to store highscore
```
Comment (tenhagena) [inline App.js:62]: Change this comment plz

File: routes/gameMap.js [hunk 2]
```diff
@@ -23,17 +23,44 @@ const styles = StyleSheet.create({
...
+    .ref('/game/')
+    .child(key)
+    .update({
+      started: true,
+    });
+};
+
 export default class GameMap extends React.Component {
   static navigationOptions = {
-    title: 'Chat with Lucy',
```
Comment (tenhagena) [inline routes/gameMap.js:28]: Why can't we keep the name this?

**Review by tenhagena — COMMENTED** [review]
Just change those two things then it LGTM (looks good to me/ looks good to merge)

File: routes/gameMap.js (hunk 2 above)
Comment (cmonteiro128) [reply routes/gameMap.js:28]: we can

**Pull request merged**